src/test/java/io/stargate/sgv2/jsonapi/service/testutil/MockAsyncResultSet.java

```diff
@@ -85,6 +85,6 @@ public ExecutionInfo getExecutionInfo() {
 
   @Override
   public boolean wasApplied() {
-    return true;
+    return rows.get(0).getBoolean("[applied]");
```

Comment:
Should this actually be `"applied"`, without brackets? At least for `InsertOperationTest` this is required.